pelita/game.py

```diff
@@ -84,13 +84,17 @@ def controller_exit(state, await_action='play_step'):
 
 def run_game(team_specs, *, max_rounds, layout_dict, layout_name="", seed=None,
              max_team_errors=5, timeout_length=3, viewers=None, controller=None, viewer_options=None,
-             store_output=False):
+             store_output=False, team_names=(None, None), allow_exceptions=False):
```

Comment:
The allow_exceptions flag we can leave for now but should probably adapt it once we know how we are going to use it.

Reply:
Agreed.